packages/core/src/components/skeleton-text/skeleton-text.tsx

```diff
@@ -13,8 +13,8 @@ import { Component, Prop } from '@stencil/core';
 })
 export class SkeletonText {
   @Prop() width = '100%';
-
+  @Prop() animated = false;
   render () {
-    return <span style={{width: this.width}}>&nbsp;</span>;
+    return <span style={{width: this.width}} class={this.animated ? 'animated' : ''}>&nbsp;</span>;
```

Comment:
Class should be like this:
```
class={'animated': this.animated}
```

Reply:
I see.. Thanks for the review, that looks cleaner :) ill update the PR later